test/e2e/network/netpol/kubemanager.go

```diff
@@ -151,89 +145,27 @@ func (k *kubeManager) getPod(ctx context.Context, ns string, name string) (*v1.P
 	return kubePod, nil
 }
 
-// probeConnectivity execs into a pod and checks its connectivity to another pod.
+// probeConnectivity execs into a pod and checks its connectivity to another pod returning the true if the connectivity state is the expected.
```

Comment:
This is incorrect; the return value tells whether the connection succeed, regardless of the expectation.
In fact, the bool return value is redundant with `err == nil`.